testChecker.cpp

```diff
@@ -0,0 +1,26 @@
+#include "testChecker.h"
+#include "checker.h"
+
+void TestVitalsMonitor::TestVitals() {
+  assert(!VitalsMonitor::vitalsOk(99, 102, 70));
+  assert(VitalsMonitor::vitalsOk(98.1, 70, 98));
+}
+
+void TestVitalsMonitor::TestBodyTemparature() {
+  assert(VitalsMonitor::bodyTemperatureOk(99));
+  assert(VitalsMonitor::bodyTemperatureOk(98.1));
+  assert(false == VitalsMonitor::isBodyTemperatureNotOk(99));
+  assert(false == VitalsMonitor::isBodyTemperatureNotOk(98.1));
+}
+
+void TestVitalsMonitor::TestPulseRate() {
+  assert(!VitalsMonitor::pulseRateOk(102));

```

Comment:
Repeating patterns in tests == an easy way to identify duplication.
If you take out the range-checking into another function, then you will need to test that functionality only once.

Reply:
Thank you. I made the changes accordingly.